client/packages/android/build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:8.9.0'
+        classpath 'com.android.tools.build:gradle:8.1.0'
```

Comment:
Can we have it matching with the version in cordova_android_plugin? 8.2.1 ~ thanks